4week/src/App.tsx

```
@@ -1,71 +1,86 @@
+                ]
+            },
+        ]
+    }
+]
+
+// protectedRoutes: 인증이 필요한 라우트
 
-export const queryClient = new QueryClient();
+const router = createBrowserRouter([...publicRoutes, ...protectedRoutes]);
 
+export const queryClient = new QueryClient({
```

ESLint

[error] Fast refresh only works when a file only exports components. Use a new file to share constants or functions between components.

(react-refresh/only-export-components)